quiet_lessons/lesson_12/task_2.py

```diff
@@ -0,0 +1,10 @@
+while True:
+    try:
+        number = int(input('Введите число: '))
+        for num_in_diapason in range(number + 1):
+            if num_in_diapason % 2 == 0:
+                print(f'{num_in_diapason} is EVEN')
+            else:
+                print(f'{num_in_diapason} is ODD')
+    except ValueError:
+        print('Некорректный тип ввода данных!')
```

Comment:
[PEP8 No newline at end of file (W292)](https://www.flake8rules.com/rules/W292.html)